src/shared/ptyfwd.c

```diff
@@ -218,15 +229,33 @@ static bool look_for_escape(PTYForward *f, const char *buffer, size_t n) {
                                 (f->escape_counter)++;
 
                                 if (f->escape_counter >= 3)
-                                        return true;
+                                        return REQUEST_EXIT;
+                        }
+
+                        break;
+                }
+
+                case 'a'...'z':
+                        if (f->escape_counter == 2) {
+                                usec_t nw = now(CLOCK_MONOTONIC);
+
+                                if (nw <= f->escape_timestamp + ESCAPE_USEC) {
+                                        f->escape_timestamp = 0;
+                                        f->escape_counter = 0;
+                                        return REQUEST_HOTKEY_BASE + *p;
+                                }
                         }
-                } else {
+
+                        _fallthrough_;
+
+                default:
                         f->escape_timestamp = 0;
                         f->escape_counter = 0;
+                        break;
```

Comment:
I do not think `break` is necessary.

Reply:
one of the static checkers doesn't like that style. insists on `_fallback_` or break...

Reply:
I'm curious which one that would be btw. We have `default: ;` in e.g. core/transaction.c and AFAIK noone has complained so far?

Reply:
hmm, dunno, let's try, dropped it now. let's see what CI says.